.gitignore

```diff
@@ -22,5 +22,8 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
-
+package-lock.json
+package.json
+node_modules
+*.env
 /*/.env
```

Comment:
@saumy-sh other than node_modules please remove all all three additions. Because of this preview deployment is failing

Reply:
I will resolve this.Any other changes that I need to do?